lms/djangoapps/shoppingcart/processors/tests/test_CyberSource2.py

```diff
@@ -41,12 +42,14 @@ class CyberSource2Test(TestCase):
     def setUp(self):
         """ Create a user and an order. """
         self.user = UserFactory()
+        self.course = CourseFactory()
         self.order = Order.get_cart_for_user(self.user)
-        self.order_item = OrderItem.objects.create(
+        self.order_item = PaidCourseRegistration.objects.create(
```

Comment:
@chrisndodge We can no longer purchase an 'OrderItem' without a course since the pdf_receipt_display_name property throws an error - no course_id field in the OrderItem model. I have modified the tests to use the PaidCourseRegistration instead or the parent OrderItem. Can you verify if these changes were correct?